assets/app/scripts/controllers/create/nextSteps.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+/* jshint unused: false */
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:CreateFinishedController
+ * @description
+ * # CreateFinishedController
+ * Controller of the openshiftConsole
+ */
+angular.module("openshiftConsole")
+  .controller("CreateFinishedController", function($scope, $http, $routeParams, DataService, $q, $location, TaskList, $parse, Navigate, $filter, imageObjectRefFilter, failureObjectNameFilter) {
+    var displayNameFilter = $filter('displayName');
+    var watches = [];
+
+    $scope.emptyMessage = "Loading...";
+    $scope.alerts = [];
+    $scope.baseUrl = location.origin;
+    $scope.projectName = $routeParams.project;
+    $scope.unfilteredBuildConfigs = {};
+    $scope.projectPromise = $.Deferred();
+    DataService.get("projects", $scope.projectName, $scope).then(function(project) {
+      $scope.project = project;
+      $scope.projectPromise.resolve(project);
+    });
+
+    var name = $routeParams.name;
+    if (name) {
+      $scope.name = name;
+
+      watches.push(DataService.watch("buildconfigs", $scope, function(buildconfigs) {
```

Comment:
assuming we are watching buildconfigs since the buildconfig we want may not yet be created when we initially navigate to the page, so a get would just 404?